tuneup.py

```diff
@@ -2,15 +2,24 @@
 # -*- coding: utf-8 -*-
 """Tuneup assignment"""
 
-__author__ = "???"
+__author__ = "iam2anangel"  # Jen Browning
 
 import cProfile
 import pstats
+import timeit
+
+#
 
 
 def profile(func):
     """A function that can be used as a decorator to measure performance"""
-    raise NotImplementedError("Complete this decorator function")
+    def wrapper():
+        prof = cProfile.Profile()
+        prof.enable()
+        prof.runcall(func)
+        prof.disable()
+      return wrapper
+
```

Comment:
This is a good start on the profiler decorator, but you'll need to also create an instance of the pstats.Stats object in order to sort and print the results of the "prof" results.  Also need to pass in *args and **kwargs to "wrapper" func and then call original func with those.  Finally also need to capture the result of func and return that from wrapper func.